components/costs/cost-usage-report.yaml

```diff
@@ -0,0 +1,43 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: setup cost usage report
+Resources:
+  CostBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: superwerker-cost-usage-report
+      LifecycleConfiguration:
+        Rules:
+          - Status: Enabled
+            ExpirationInDays: 1095
+
+  CostBucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
```

Comment:
The value (same goes for the string values below) likely does _not_ need escaping